### PR TITLE
DOC-3203 Word cloud: reflect new instructions field

### PR DESCRIPTION
**Pull request by catong**
# [DOC-3203](https://openedx.atlassian.net/browse/DOC-3203)

This PR updates the B&R word cloud topic to reflect the addition of a new "Instructions" field in word cloud settings. 
I kept the existing example word cloud image but cropped out the heading displays.
### Reviewers
- [x] Dev/SME: @clrux 
- [ ] T&L: @cahrens 
- [x] Doc: @lamagnifica or @srpearce or @pdesjardins 
- [x] Product: @sstack22 

FYI: @jaakana
### Sandbox (from AC-552)

Studio: https://studio-clrux-ac-551.sandbox.edx.org/container/block-v1:edX+DemoX+Demo_Course+type@vertical+block@3785b1366e7749edacf28d5f7dc6f98a

LMS: https://clrux-ac-551.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/courseware/9fca584977d04885bc911ea76a9ef29e/07bc32474380492cb34f76e5f9d9a135/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%403785b1366e7749edacf28d5f7dc6f98a
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

**Comment by downzer0** [comment]
When it's ready 👍

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 1]
```diff
@@ -6,14 +6,14 @@ Word Cloud Tool
...
 .. note:: EdX offers provisional support for this tool.
 
-In a word cloud, students enter words into a field in response
-to a question or prompt. The words that all of the students enter then
+In a word cloud, learners enter words into a field in response
+to a question or prompt. The words that all of the learners enter then
 appear instantly as a colorful graphic, with the most popular responses
-appearing largest. The graphic becomes larger as more students answer.
-Students can both see the way their peers have answered and contribute
+appearing largest. The graphic becomes larger as more laerners answer.
```
Comment (catong) [inline en_us/shared/exercises_tools/word_cloud.rst]: Oops a typo: I'll fix.

**Comment by sstack22** [comment]
👍 :)

**Comment by catong** [comment]
@clrux Could you please take a second look at the revised settings descriptions? Updated after final design decisions made on your PR.

**Comment by downzer0** [comment]
👍

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 2]
```diff
@@ -6,15 +6,14 @@ Word Cloud Tool
 
 .. note:: EdX offers provisional support for this tool.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/word_cloud.rst:7]: still provisional ^ ?

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 3]
```diff
@@ -6,15 +6,14 @@ Word Cloud Tool
...
+graphic becomes larger as more learners answer. Learners can see how their
+peers answered as well as contribute their thoughts to the group.
 
-For example, the following word cloud was created from students'
-responses to a question in a HarvardX course.
+For example, the following word cloud was created from learners' responses to
+a question about why they think addressing climate change is difficult.
 
 .. image:: ../../../shared/images/WordCloudExample.png
   :alt: Image of a word cloud problem.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/word_cloud.rst]: nice to have: better description of the image: what answers appear largest? about how many answers are depicted?

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 4]
```diff
@@ -45,16 +44,28 @@ To create a word cloud, follow these steps.
...
+     below the name that you give the word cloud. It is important to provide
+     clear instructions, especially for learners who have accessibility
+     requirements.
+
+   * **Maximum Words**: The maximum number of words that the generated word
+     cloud displays. If learners enter 300 different words but the maximum
+     is set to 250, only the 250 most commonly entered words appear in the
+     word cloud.
+
+   * **Show Percents**: The number of times that learners have entered a given
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/word_cloud.rst]: nice to have: is this a true/false question? rephrase accordingly: When set to true (or yes or whatev), the number of times...

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 5]
```diff
@@ -45,16 +44,28 @@ To create a word cloud, follow these steps.
...
+   * **Inputs**: The number of text boxes provided for learners to enter words,
      phrases, or sentences.
-   * **Maximum Words**: The maximum number of words that the word cloud
-     displays. If students enter 300 different words but the maximum is set to
-     250, only the 250 most commonly entered words appear in the word cloud.
-   * **Show Percents**: The number of times that students have entered a given
+
+   * **Instructions**: Instructions that give learners specific information
+     about how to contribute to this word cloud. These instructions appear
+     below the name that you give the word cloud. It is important to provide
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/word_cloud.rst]: the ^display name

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 6]
```diff
@@ -45,16 +44,28 @@ To create a word cloud, follow these steps.
...
 #. In the component that appears, select **Edit**.
-#. In the component editor, specify the settings that you want. You can
-   leave the default value for everything except **Display Name**.
+#. In the component editor, specify these settings.
 
    * **Display Name**: This name appears as a heading above the problem.
-   * **Inputs**: The number of text boxes into which students can enter words,
+     Replace the default name with a specific title or name for your word
+     cloud. If you do not replace the default name, the heading "Word Cloud"
+     is used.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/word_cloud.rst]: I'd suggest replacing the 2nd sentence with something like, "Unique, descriptive display names help you and your learners identify problems quickly and accurately." (from create_problem.rst) Currently, it's somewhat redundant with "Replace the default name..."

File: en_us/shared/exercises_tools/word_cloud.rst (hunk 2 above)
Comment (catong) [reply en_us/shared/exercises_tools/word_cloud.rst:7]: Yes

File: en_us/shared/exercises_tools/word_cloud.rst [hunk 7]
```diff
@@ -45,16 +44,28 @@ To create a word cloud, follow these steps.
...
-   * **Maximum Words**: The maximum number of words that the word cloud
-     displays. If students enter 300 different words but the maximum is set to
-     250, only the 250 most commonly entered words appear in the word cloud.
-   * **Show Percents**: The number of times that students have entered a given
+
+   * **Instructions**: Instructions that give learners specific information
+     about how to contribute to this word cloud. These instructions appear
+     below the name that you give the word cloud. It is important to provide
+     clear instructions, especially for learners who have accessibility
+     requirements.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/word_cloud.rst]: Would an example be possible here? I'm not actually able to imagine what instructions might be helpful.

**Comment by lamagnifica** [comment]
a few suggestions to consider, 👍

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1158

**Pull request merged**